test/api/application/bucket_test.go

```diff
@@ -0,0 +1,26 @@
+package application
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/client"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestApplicationBucket(t *testing.T) {
+	// Test Facts subresource on the first sample application only.
+	application := Samples()["Minimal"]
+
+	// Create the application.
+	assert.Must(t, Create(&application))
```

Comment:
So, here we use the helper function, but in the create_get_delete_test.go we use the raw client. Why?

Reply:
Focus in this test file was on the buket get, so it looked to me easier to use short command to create a application for the bucket test. (this is not part of the reuse test pkg, it is its usage within the API test, so it is expected that test author might decide what fits better for its use case, more desciption in part of https://github.com/konveyor/tackle2-hub/pull/268#issuecomment-1520072758)